docs/en/stack/security/get-started-security.asciidoc

```diff
@@ -0,0 +1,311 @@
+[role="xpack"]
+[[security-getting-started]]
+== Getting started with security
+
+In this tutorial, you learn how to secure a cluster by configuring users and 
+roles for use in {es}, {kib}, {ls}, and {metricbeat}. 
+
+[float]
+[[gs-security-prereqs]]
+=== Before you begin
+
+Install and start {es}, {kib}, {ls}, and {metricbeat} as described in 
+<<get-started-elastic-stack>>. 
+
+Launch the {kib} web interface by pointing your browser to port 5601. For 
+example, http://127.0.0.1:5601[http://127.0.0.1:5601].
+
+[float]
+[[gs-security-license]]
+=== Install a trial license
+
+When you installed the products, they applied Basic licenses with no expiration 
+dates. To view your license in {kib}, go to **Management** and click 
+**License Management**.
+
+[role="screenshot"]
+image::images/management-license.png["License management screenshot in Kibana"]
+
+To secure your cluster, you must obtain a license that includes the {security} 
+features. For more information about Elastic license levels, see 
+https://www.elastic.co/subscriptions.
+
+You can start a 30-day trial to try out all of the Platinum features, including 
+{security}. Click **Start trial**.
+
+IMPORTANT: If your cluster has already activated a trial license for the current 
+major version, you cannot start a new trial. For example, if you have already 
+activated a trial for v6.0, you cannot start a new trial until v7.0.
+
+At the end of the trial period, your license reverts to a Basic license. If you 
```

Comment:
Is this correct?
I didn't think we reverted automatically, but I may have missed something.

Reply:
I don't think it is correct either; the license expires and elasticsearch may operate in a degraded state if security is enabled. That is stats and health apis are not available.